application/src/main/java/org/togetherjava/tjbot/logging/discord/DiscordLogForwarder.java

```diff
@@ -169,13 +176,13 @@ private static LogMessage ofEvent(LogEvent event) {
             Instant timestamp = Instant.ofEpochMilli(event.getInstant().getEpochMillisecond());
 
             WebhookEmbed embed = new WebhookEmbedBuilder()
-                .setAuthor(new WebhookEmbed.EmbedAuthor(authorName, null, null))
+                .setAuthor(new WebhookEmbed.EmbedAuthor(authorName, null,
+                        linkToSource(event.getSource().getClassName(), sourceCodeBaseUrl)))
```

Comment:
The linkToSource thing should be moved out of setAuthor like the other embed properties to improve readability